openapi/anchor-platform/bundled_callback.yml

```diff
@@ -711,6 +731,335 @@ components:
+          $ref: '#/components/schemas/Amount'
+        amount_out:
+          $ref: '#/components/schemas/Amount'
+        amount_fee:
+          $ref: '#/components/schemas/Amount'
+        quote_id:
+          type: string
+        started_at:
+          type: string
+          format: date-time
+        updated_at:
+          type: string
+          format: date-time
+        completed_at:
+          type: string
+          format: date-time
+        transfer_received_at:
+          type: string
+          format: date-time
+        message:
+          type: string
+        refunds:
+          $ref: '#/components/schemas/Refunds'
+        stellar_transactions:
+          type: array
+          items:
+            $ref: '#/components/schemas/StellarTransaction'
+        external_transaction_id:
+          type: string
+        customers:
+          type: object
+          description: |
+            The Identification info of the sending and receiving customers. If they were created through [SEP-12](https://stellar.org/protocol/sep-12),
+            this object should contain the SEP-12 customer `id`. Otherwise, the `account` address of the customer.
+          properties:
+            sender:
+              $ref: '#/components/schemas/StellarId'
+            receiver:
+              $ref: '#/components/schemas/StellarId'
+        creator:
+          $ref: '#/components/schemas/StellarId'
+    Quote:
+      type: object
+      properties:
+        id:
+          type: string
+        sell_amount:
+          type: string
+        sell_asset:
+          type: string
+        buy_amount:
+          type: string
+        buy_asset:
+          type: string
+        expires_at:
+          type: string
+          format: date-time
+        price:
+          type: string
+        total_price:
+          type: string
+        creator:
+          $ref: '#/components/schemas/StellarId'
+        transaction_id:
+          type: string
+        created_at:
+          type: string
+          format: date-time
+    Event:
+      type: object
+      properties:
+        id:
+          type: string
+        type:
+          type: string
+          enum:
+            - transaction_created
+            - transaction_status_changed
+            - transaction_error
+            - quote_created
+          description: |
+            The transaction event type. Can be one of the following:
+            - `transaction_created` - a transaction was created through the SEP endpoints. The payload is in the `transaction` field.
+            - `transaction_status_changed` - the status of a transaction has changed. The payload is in the `transaction` field.
+            - `transaction_error` - error processing the transaction. The payload is in the `transaction` field.
+            - `quote_created - a quote was created via the SEP38 API. The payload is in the `quote` field.
+        timestamp:
+          type: string
+          format: date-time
+        payload:
```

Comment:
Ideally payload should be
```
oneOf
   - PayloadTransaction
   - PayloadQuote
 
PayloadTransaction:
oneOf
   - PayloadTransactionSep24 ( object is transaction: TransactionSEP24 )
   - PayloadTransactionSep31 (object is transaction: TransactionSEP31 )
```

But types in openapi is very tedious so idk if it's worth to even bother

Reply:
We don't have to decide on this now but I actually don't know if creating different permutations of the platform transaction schema is the right approach, as opposed to having a unified platform transaction schema documented. I think this makes a bit more sense, even if we have to explain that the value of `transaction.sep` has an impact on the expected values for `transaction.status` and `transaction.kind`.

Reply:
I think having separate types between SEPs make sense to indicate mandatory/optional/not used fields. Everything else is too much work to clearly separate types.

Reply:
I don't think the fields differ between SEPs though, other than the values of `status` and `kind` right?

Reply:
No, there are quite a few differences in platform transaction between sep-24 and sep-31

Reply:
Lets merge this as-is but we can continue to discuss this. The intention behind the platform API was to unify the schema across SEPs so if there are schema differences we should resolve them.

Reply:
![image](https://github.com/stellar/stellar-docs/assets/62526399/738606cb-5f61-4df8-bcad-cb588a5300c6)
![image](https://github.com/stellar/stellar-docs/assets/62526399/91838d04-7707-41f9-928d-2838d0957904)

Reply:
Seems like the diff is:

- `kind` has different values depending on the value of `sep`
- `status` has different values depending on the value of `sep`
- `quote_id` is present for SEP-31 transactions but not for SEP-24.
  - this should be present on SEP-24 though, since its supported in the protocol, no?
- `source_account` and `destination_account` is present on SEP-24, but not SEP-31
  - I don't see a reason why they shouldn't be on SEP-31
- `memo` and `memo_type` is on SEP-24 but not SEP-31
  - same here, not sure why we don't have this on SEP-31
- `customers` is on SEP-31, not SEP-24
  - we've discussed this before, I think the outcome was to add `customers` to SEP-24 but sender and receiver will be the same
- `creator` is on SEP-31, not SEP-24
  - this actually seems like a bug, how do businesses know which wallet created the transaction without this?

In every case above, there doesn't seem to be a problem making the schema's match.